res/database/Default/user/sjaele.xml

```diff
@@ -975,11 +1121,13 @@
 		<news id="Sjaele-news-Pfeifenbläser-angeklagt" type="0" creator="7430" created_by="Sjaele">
 			<title>
 				<de>Pfeifenbläser angeklagt</de>
-				<en>Whistleblowers indicted</en>
+				<en>Whistleblowers indicted</en><!--Unclear to me. It is about the "whistleblower" , that disturbs, whistles ?-->
```

Comment:
remove, if clear now or move the comment to the pl line

Reply:
Done.